prod/api/projects_list_api.py

```diff
@@ -21,8 +22,16 @@ class ProjectsListResource(Resource):
 
     @ns.response(200, 'Success', fields.List(fields.Nested(code_20x_swg)))
     def get(self):
-        response_object =\
-            [project.serialize() for project in ProjectDBModel.query.all()]
+        type = request.args.get("type")
+        if type:
+            for item in ProjectTypeEnum:
+                if item.value == type:
+                    enumType = item
```

Comment:
¿Esta variable no deberia esta declarada fuera del Scope de esta ciclo for?
Asumo que funciona por Magia de Python